main.sh

```diff
@@ -30,13 +30,14 @@ LOG_LEVEL="${LOG_LEVEL:-6}" # 7 = debug -> 0 = emergency
 # Commandline options. This defines the usage page, and is used to parse cli
 # opts & defaults from. The parsing is unforgiving so be precise in your syntax
 read -r -d '' usage <<-'EOF'
-  -f   [arg] Filename to process. Required.
-  -t   [arg] Location of tempfile. Default="/tmp/bar"
-  -d         Enables debug mode
-  -h         This page
+  -f --file  [arg] Filename to process. Required.
+  -t --temp  [arg] Location of tempfile. Default="/tmp/bar"
+  -v               Enable verbose mode, print script as it is executed
+  -d --debug       Enables debug mode
+  -h --help        This page
 EOF
 
-# Set magic variables for current file and its directory. 
+# Set magic variables for current file and its directory.
```

Comment:
Yes of course : ) I think I have turned it off in my editor for markdown trailing space preservation. I should really make a case-by-case exception going forward :smile: